crates/sui-core/src/transaction_input_checker.rs

```diff
@@ -190,7 +190,7 @@ async fn check_gas(
         let cost_table = SuiCostTable::new(protocol_config);
         let storage_gas_price = protocol_config.storage_gas_price();
 
-        // TODO: We should revisit how we compute gas price and compare to gas budget.
+        // MUSTFIX: We should revisit how we compute gas price and compare to gas budget.
```

Comment:
We compute the max of computation_gas_price and storage_gas_price here too:https://github.com/MystenLabs/sui/blob/28526f64b170d2dbc4bf1c9660fad05858f8b6b8/crates/sui-types/src/gas.rs#L368
which seems to be used for programmable txns.

Reply:
the plan is to remove those and to use computation for computation and gas for gas...
just fyi

Reply:
yup just pointing out another place where we should considering adding a MUSTFIX...